src/sentry/api/endpoints/project_filters.py

```diff
@@ -1,18 +1,37 @@
+from drf_spectacular.utils import extend_schema
 from rest_framework.request import Request
 from rest_framework.response import Response
 
+from sentry.api.api_owners import ApiOwner
 from sentry.api.api_publish_status import ApiPublishStatus
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.serializers.models.project_filters import ProjectFiltersResponse
+from sentry.apidocs.constants import RESPONSE_BAD_REQUEST, RESPONSE_UNAUTHORIZED
+from sentry.apidocs.parameters import GlobalParams
+from sentry.apidocs.utils import inline_sentry_response_serializer
 from sentry.ingest import inbound_filters
 
 
+@extend_schema(tags=["Projects"])
 @region_silo_endpoint
 class ProjectFiltersEndpoint(ProjectEndpoint):
+    owner = ApiOwner.UNOWNED
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PUBLIC,
     }
 
+    @extend_schema(
+        operation_id="List a Project's Filters",
+        parameters=[GlobalParams.ORG_ID_OR_SLUG, GlobalParams.PROJECT_ID_OR_SLUG],
+        responses={
+            200: inline_sentry_response_serializer(
+                "ProjectFiltersResponse", ProjectFiltersResponse
```

Comment:
AFAICT an ad-hoc `TypedDict` is appropriate here. Let me know if there's a better convention to follow for naming and/or where to define it.